src/main/proto/management.proto

```diff
@@ -80,19 +80,24 @@ message EdgeLabel {
     google.protobuf.Int64Value id = 1;
     string name = 2;
     repeated EdgeProperty properties = 3;
+    enum Directed {
```

Comment:
This already done with  Direction but directed would be a better name

Reply:
Yea. And maybe we can eliminate Direction all together?  What do you think?